projects/README.md

```diff
@@ -0,0 +1,65 @@
+# Aframe Projects
+Each aframe project represents an individually containerized sub-task of the aframe analysis. This allows project environments
+to be lightweight, flexible and portable. Projects are meant to produce _artifacts_ of some specific experiment. By artifact, we typically mean some file (e.g training data, optimized models, analysis plots, etc.) saved to disk somewhere. Projects should be kept modular and specific to the artifact they are designed to generate.
+
+
+## Building Project Containers
+Most projects are fully python based, and their environments are managed using [poetry](https://python-poetry.org/).
+Some projects also require the use of [`Mamba`](https://mamba.readthedocs.io/en/latest/).
+
+In the root directory of (most) projects is an `apptainer.def` file that containerizes 
+the project application using [Apptainer](https://apptainer.org/docs/user/latest/) (formerly known as Singularity).
+
+Before building project container images, it is recommended that you set the `AFRAME_CONTAINER_ROOT` environment variable.
+A good location for this would be in your `~/.bash_profile` (or whichever shell-dependent scripts are automatically run when you login)
+
+```bash
+echo export AFRAME_CONTAINER_ROOT=~/aframe/images/ >> ~/.bash_profile
+mkdir -p $AFRAME_CONTAINER_ROOT
+```
+
+This is the location where `aframe` images will be stored. `luigi`/`law` tasks will look for images in this location.
+
+As an example, let's build the `data` container image. Once inside the `data` root directory, building the image is as simple as 
+
+```bash
+apptainer build $AFRAME_CONTAINER_ROOT/data.sif apptainer.def
+```
+
+> **_Note It is highly recommended that you name containers after the corresponding project. Although not strictly necessary, this is the default behavior in the `luigi`/`law` tasks_**
+
+## Executing a Container
+Most projects come with a command line interface (CLI) built with the extremely flexible [jsonargparse](https://jsonargparse.readthedocs.io/en/stable/). As an example of how you might run a command inside the container, let's use the CLI in the data container to query for science-mode segments from (gwosc)[https://gwosc.org/]
+
+```bash
+mkdir ~/aframe/data/
+apptainer run ~/aframe/images/data.sif \
```

Comment:
For consistency, this should be `apptainer run $AFRAME_CONTAINER_ROOT/data.sif \`